### ISSUE TITLE
Weird floating nested guy

### ISSUE THREAD

**Issue by gitbirb** [cmss13-devs/cmss13#4561]
### Testmerges

#4505: 'Get Flat Icon Fixes' by Drulikar at commit 602124566b #4391: 'Hijack objectives' by morrowwolf at commit c34cb004ab #4534: '"Fixes" missing window frame icon states' by morrowwolf at commit 7cf004c46a #4535: 'Floodlight rework' by morrowwolf at commit 311b5109a8 #4549: 'Helmet Optics Update' by morrowwolf at commit a7820b1907 #4550: 'Xeno structure cheese reduction' by morrowwolf at commit 54f11548c5

### Description of the bug

I saw this midround on LV-522: Chance's Claim. No clue what causes it. The nest is not near any walls whatsoever, but still nested.
![image](https://github.com/cmss13-devs/cmss13/assets/44538569/3d65bf75-6a15-4bd0-8f53-215c67d3de66)

### What's the difference with what should have happened?

Nests should be on walls.

### How do we reproduce this bug?

1. no clue how to reproduce
2. 
3. Saw it on LV-522
...
Round ID was 18881 if that helps.

### Issue Bingo

- [ ] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by TeDGamer** [comment]
I think one way to reproduce was to have shaking marines before nesting.

Coffee or neurotoxin.

I noticed it happened once when a marine was shaking then nested sometimes they pop out like that

**Comment by gitbirb** [comment]
> I think one way to reproduce was to have shaking marines before nesting.
> 
> Coffee or neurotoxin.
> 
> I noticed it happened once when a marine was shaking then nested sometimes they pop out like that

I think that has to do more with #4547 then this one. In that issue, the nest is still on the wall, but the person is floating away from the nest. In this issue, the nest is floating, but the person is secured in the nest with correct position.

**Comment by morrowwolf** [comment]
Yeah this is probably due to setting initial() x and y offsets somewhere but the issue is the entire thing needs changed so we have a list of things offsetting our mob with references to what has offset it so when one offset stops we recalculate our total offset rather than set back to original offset. (This is when I looked into this like six months ago so I may be misremembering).

**Comment by Huffie56** [comment]
![image](https://github.com/cmss13-devs/cmss13/assets/117036822/f9158f92-895f-4e5a-a07a-79bb0b352a65)
was it the location?
do you have any picture that isn't that much zoomed in?

**Comment by Huffie56** [comment]
![image](https://github.com/cmss13-devs/cmss13/assets/117036822/aa568b35-ec95-4267-b09c-4a4be843fc06)
confirmed bug it's a bug in the code when their is no wall to attach to...

**Comment by Huffie56** [comment]
https://discord.com/channels/150315577943130112/745447048261795890/1184126587722018987
